src/core/sensor/qgssensormanager.cpp

```diff
@@ -147,6 +148,15 @@ void QgsSensorManager::captureSensorData()
   emit sensorDataCaptured( sensor->id() );
 }
 
+void QgsSensorManager::handleSensorErrorOccurred( const QString & )
```

Comment:
Would it make sense to send the sensor instead of its id (to avoid the extra lookup from id back to sensor later on)?

Reply:
I'd prefer to keep it as is for now to match other signals (which work nicely with the sensor model class).